paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/UpdatePaymentMethodInteractor.kt

```diff
@@ -181,7 +182,9 @@ internal class DefaultUpdatePaymentMethodInteractor(
             cardBrandFilter = cardBrandFilter,
             onBrandChoiceChanged = onBrandChoiceSelected,
             areExpiryDateAndAddressModificationSupported = isModifiable && canUpdateFullPaymentMethodDetails,
-            addressCollectionMode = addressCollectionMode,
+            billingDetailsCollectionConfiguration = BillingDetailsCollectionConfiguration(
+                address = addressCollectionMode
+            ),
```

Comment:
@toluo-stripe Updated the `CardEdit` Form to accept contact info (email, name, phone) if you can take a look! 

Also could you confirm the `BillingDetailsCollectionConfiguration` we want to pass outside of Link?